backend/index.js

```diff
@@ -32,6 +32,9 @@ app.set('view engine', 'njk')
 
 nunjucksSetup(app, path)
 
+app.locals.phaseName = config.phaseName
```

Comment:
Could we put this into it's own setup function? 

Something along the lines of: 
`app.use(setupPhase??())`   

That way it'll be consistent with everything else in this file.